Lesson20/Kadir/app.js

```diff
@@ -9,25 +9,51 @@ form.addEventListener('submit', (event) => {
   // Create a new task item and add it to the list
 
   const listItemElement=document.createElement("li");
-  listItemElement.innerText = input.value;
+  const spanElement = document.createElement("span");
+  spanElement.innerText = input.value;
+  listItemElement.appendChild(spanElement);
   
   // Remove button
   const removeButton = document.createElement("button");
   removeButton.innerText = 'X';
   // TODO: Update styles of removeButton! (use button--light class)
-  listItemElement.appendChild(removeButton);
-
+  
```

Comment:
TODO: Update styles of removeButton! (use button--light class)TODO: Update styles of removeButton! (use button--light class)

This requirement is still missing, please add styles to the delete button

Reply:
It  is in row 29 already.